assets/js/hooks/session.js

```diff
@@ -1032,6 +1036,7 @@ const Session = {
 
   setFocusedEl(focusableId, { scroll = true, focusElement = true } = {}) {
     this.focusedId = focusableId;
+    this.el.dataset.focusedId = focusableId;
```

Comment:
You can do `this.el.setAttribute("data-js-focused-id", focusableId);`. We make sure that `data-js-` is not overridden by LV, so we don't need to duplicate in `update()` :)

Also, for consistency let's remove the attribute if `focusableId` is `null`.